packages/preview/polylux-0.2.0/typst.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "polylux"
+version = "0.2.0"
+entrypoint = "polylux.typ"
+authors = ["Andreas Kröpelin", "contributors"]
+license = "MIT"
+description = "Presentation slides creation with Typst"
+repository = "https://github.com/andreasKroepelin/polylux"
+exclude = ["README.md", "LICENSE"]
```

Comment:
I'm not sure it matters much, but I'd rather keep the LICENSE so that it is always shipped alongside the code. All other packages also ship the README, but that matters less. 

I don't think its problematic to ship both though, it won't bloat the tar.gz bundle much. The exclusion mechanism was intended mostly for heavy things like images and PDF manuals.